### ISSUE TITLE
Ability to disable terminal escape sequences (ie. for color)

### ISSUE THREAD

**Issue by p4block** [bluenviron/mediamtx#1477]
## Describe the feature
If the thing used to display the stdout can't parse escape sequences you get very ugly printing
![image](https://user-images.githubusercontent.com/5960575/219354866-d6039ee7-788d-4551-833b-601f5772b4fe.png)

Some software allows enabling "simple output" for these constrained environments

**Comment by rgl** [comment]
@aler9 should color be removed altogether or should it respect the [`NO_COLOR` environment variable](https://no-color.org/)?

**Comment by aler9** [comment]
@rgl based on the page you posted and additional documentation i gathered, color should be removed when either

* `NO_COLOR` is set
* `TERM` is not `xterm-256color`

regarding the last one, i have to perform a on Windows, then we can implement this feature.

**Comment by aler9** [comment]
@rgl it seems that the underlying color library (https://github.com/gookit/color) already checks for `NO_COLOR`, maybe we can check for `TERM` on Linux / Darwin and for another variable on Windows...

**Comment by aler9** [comment]
i looked at similar software and these are the conditions to disable colored output:

* `NO_COLOR` is set
* output is not a terminal

the second condition is missing and is implemented with #2050

**Comment by rgl** [comment]
@aler9 oh, when I wrote about `NO_COLOR=1`, I didn't actually try it, because this issue was still open, I've assumed there still was no way to disable the color output, so I decided to mention the `NO_COLOR=1` way, with the intent of starting the conversation to try to fix it myself.

I've now finally used the `NO_COLOR=1` environment variable and it works as expected. :-)

Having it detect the terminal is nice too! Thanks for implementing it!
